crates/conformance-test-utils/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+pub mod kakarot;
+
+use serde::{Deserialize, Serialize};
+use serde_with::serde_as;
+use starknet::core::serde::unsigned_field_element::UfeHex;
+use starknet::core::types::FieldElement;
+
+#[serde_as]
+#[derive(Serialize, Deserialize)]
+pub struct Felt(#[serde_as(as = "UfeHex")] pub FieldElement);
```

Comment:
can you remind me why we need this?

Reply:
Sorry I missed this when I was cleaning up, will remove it 👍